### ISSUE TITLE
phpMyAdmin - Configuring pmadb failed to access 

### ISSUE THREAD

**Issue by felixrauhaa** [phpmyadmin/phpmyadmin#17710]
first of all, i have been working with xampp for a long time and now with phpmyadmin version 5.2.0. 
The websites on my localhost work.
I can create new tables, export, import everything ok.

Now since yesterday have an error message at localhost/phpmyadmin :

Der phpMyAdmin-Konfigurationsspeicher ist nicht vollständig konfiguriert, einige erweiterte Funktionen wurden deaktiviert. [Finden Sie heraus warum](http://localhost/phpmyadmin/index.php?route=/check-relations).
Oder wechseln Sie in einer beliebigen Datenbank zum Tab „Operationen“, um die Einstellung dort vorzunehmen.

Unfortunately I can't get anywhere with this error, even after intensive google search.
I have fixed the mysql table, both the phpmyadmin table does not show any error either.

When I go to a table "operations" after instruction, it shows me:
Der phpMyAdmin Konfigurations-Speicher wurde deaktiviert. [Finden Sie heraus warum].

Then again when I go to this link it shows me:
phpMyAdmin configuration storage Dokumentation
Konfiguration des pmadb… fehlerhaft Dokumentation

Allgemeine Verknüpfungsfunktionen Deaktiviert

 Erzeuge die PhpMyAdmin-Konfigurationsspeicherung in der aktuellen Datenbank.

When I go to "Create" it displays with this error:

#1044 - Access denied for user 'pma'@'localhost' to database 'horb'.
phpMyAdmin configuration storage documentation
Configuration of pmadb... incorrect Documentation

General linking functions disabled

 Create the PhpMyAdmin configuration storage in the current database.

I can't get anywhere with this at the moment! Does anyone have an idea or solution?

Thanks a lot

Regards 
Felix

**Comment by williamdes** [comment]
Hi @felixrauhaa 

Could you try the latest 5.2 version in development [(phpMyAdmin 5.2+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.2+snapshot) ?
I am not sure what is is the error on your end.

**Comment by williamdes** [comment]
Hi Felix,

> The error is still displayed, see screenshot....
I don't know what to set now.  ??

By email response, attachements do not work
Can you post the screenshot on https://github.com/phpmyadmin/phpmyadmin/issues/17710 ?

**Comment by felixrauhaa** [comment]
Hi William,

here 
![phpmyadmin](https://user-images.githubusercontent.com/112874469/189105232-c8426de3-136f-4e5f-9507-5ec6a854be09.jpg)
the Screenshot....

Regards

Felix

**Comment by williamdes** [comment]
Hi @felixrauhaa
Thank you for the screenshot, I understand better the error you are having 
Do you have any specific config for pmadb on your configuration file?

**Comment by felixrauhaa** [comment]
Hi william,

I have not changed anything, nor created a spec. configuration, see screenshot of config.inc.php....

Regards,

Felix
![config_inc_php](https://user-images.githubusercontent.com/112874469/189898458-e9d52ac5-91e4-4585-b549-5713e8e49ea4.jpg)

**Comment by williamdes** [comment]
Hi @felixrauhaa
Your config file is missing a password for controlpass.
Docs: https://docs.phpmyadmin.net/en/latest/config.html#cfg_Servers_controlpass

I am not sure why it is missing a value in your file. Does your user have access to the phpmyadmin database?

**Comment by felixrauhaa** [comment]
Hi William,

yes, I have access to the phpmyadmin database.
I have also checked the databases mysql and phpmyadmin and repaired the errors if necessary.

The other problem is that mysql keeps crashing.
I solve the problem by renaming the folder data in msql/, e.g. data_old - then copy the folder backup and rename it data - then I copy all the databases I have created (except mysql, performance_schema, phpmyadmin and test) from data_old and copy ibdata1 into the new folder data. Then I restart Xampp. Then everything runs cleanly again...
But this is very annoying over time...
This has been going on for over 2-3 months now.

Regards Felix
